jekyll/_cci2/language-android.md

```diff
@@ -133,12 +147,107 @@ and macOS capabilities. Please check out [this example React Native
 application](https://github.com/CircleCI-Public/circleci-demo-react-native)
 on GitHub for a full example of a React Native project.
 
-## Disabling Pre-Dexing to Improve Build Performance
+## Testing With Firebase Test Lab
+
+To use Firebase Test Lab with CircleCI,
+first complete the following steps.
+
+1. **Create a Firebase project.**
```

Comment:
I don't think these steps need bold font.

Reply:
I stole this stylistic choice from [Google's documentation](https://firebase.google.com/docs/test-lab/android/continuous). I like how they bold the task itself, then provide more details in non-bold.